docs/src/main/sphinx/connector/sql-support-others.fragment

```diff
@@ -0,0 +1,14 @@
+Other supported commands:

```

Comment:
Give this fragment a meaningful anem. "sql-support-others" is not very meaningful.

Maybe "sql-supported-statements-not-connector-specific"